CHANGELOG.md

```diff
@@ -1,5 +1,23 @@
 # Change Log
 
+## [v1.0.1](https://github.com/eclipse-iceoryx/iceoryx/tree/v1.0.0) (2021-06-15)
+
+[Full Changelog](https://github.com/eclipse-iceoryx/iceoryx/compare/v1.0.0...v1.0.1)
```

Comment:
When I click on this link I only see the message: There isn’t anything to compare.
But the release 1.0 compare works. Is it maybe missing?

Reply:
Yes the corresponding git tag will be set after merge